introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi there, Dojo. I am Jay, and I'm at your service. Give me all I can handle!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?